src/wallet/rpcwallet.cpp

```diff
@@ -2840,79 +2840,6 @@ UniValue mintzerocoin(const UniValue& params, bool fHelp)
 
 }
 
-UniValue mintzerocoinV3(const UniValue& params, bool fHelp)
```

Comment:
Agree. Broken functions

Reply:
Guys do we need to delete this? I can try to fix them. We don't have any RPC functions to substitute these right?

Reply:
we use `mint` instead.

Reply:
Exactly. `mint` and `spendmany` are enough